src/cloud/live/sens-data-over.md

```diff
@@ -6,202 +6,132 @@ functional_areas:
 
    ```bash
-   ssh <SSH URL> "php vendor/bin/m2-ece-scd-dump"
+   ssh <SSH-URL> "./vendor/bin/ece-tools config:dump"
    ```
 
-### Step 2: Transfer and add the file to Git {#transfer-file}
+{:.procedure}
+To transfer the configuration file and apply to another environment:
 
-Push the `config.php` file to Git. To push this file to the `master` Git branch, you need to complete a few extra steps because this environment is read-only.
+1. On your local workstation, change to the Cloud project root directory.
 
-1. Transfer the `config.php` file to your local system using `rsync` or `scp`. You can only add this file to the Git branch through your local.
+1. Transfer the `/app/etc/config.php` file to your local project folder using `rsync` or `scp`.
 
    ```bash
-   rsync <SSH URL>:app/etc/config.php ./app/etc/config.php
+   rsync <SSH-URL>:app/etc/config.php ./app/etc/config.php
    ```
 
-1. Add and push the `config.php` file to the Git master branch.
+1. Add, commit, and push code changes to update a remote environment.
 
    ```bash
-   git add app/etc/config.php && git commit -m "Add system-specific configuration" && git push origin master
+   git add app/etc/config.php
    ```
 
-When you add the `config.php` file to Git, all build and deploy processes move static content deployment (SCD) to the _build_ phase. The method for the deployment uses the scope. The default option is [`quick`]({{ site.baseurl }}/guides/v2.3/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick). You can change the strategy by setting an environment variable for [`SCD_STRATEGY`]({{ site.baseurl }}/cloud/env/variables-deploy.html).
-
- {:.bs-callout-info}
-Once you add this file to your code, you should not delete it. If you need to remove or edit settings, you must manually edit the file to make changes.
-
-### Step 3 & 4: Push Git branch to Staging and Production {#push-git}
-
-Log in to the Magento Admin panel in those environments to verify the settings. Only configured settings display when you use the `scd-dump` command. Continue configuring the environment, if needed.
+   ```bash
+   git commit -m "Add system-specific configuration"
+   ```
 
-For Starter, when you push, the updated code pushes to the active environment. Merge the branch to Staging and Production. Complete any additional configurations in Staging and Production as needed.
+   ```bash
+   git push origin <branch-name>
+   ```
 
-For Pro, when you push to the Git branch, the Integration `master` environment updates. Deploy this branch to Staging and Production. Complete any additional configurations in Staging and Production as needed.
+After the deployment is complete, log in to the Magento Admin panel for the updated environment to verify the settings. Only configured settings display when you use the dump command. Continue to merge any additional configurations to the Staging and Production environments, as needed.
```

Comment:
" Only configured settings display when you use the dump command."  --- this seems unclear.   Display where? Settings configured in config.php file?   Are the other fields blank. If not configured, does the setting use default value?